ApiLoader.lua

```diff
@@ -76,26 +76,37 @@ local g_CTypeToLuaType =
 	["Vector3<int>"]    = "Vector3i",
 	["Vector3<float>"]  = "Vector3f",
 	["Vector3<double>"] = "Vector3d",
+	["const Vector3<int>"]    = "Vector3i",
+	["const Vector3<float>"]  = "Vector3f",
+	["const Vector3<double>"] = "Vector3d",
 }
 
 
 
 
 
+--- Converts C++ types to their Lua counterparts.
+-- modifies a_Signature directly, but also returns it afterwards.
```

Comment:
What exactly is `a_Signature`'s structure? Describe it, because I think the term is not used exactly in the same sense as everywhere else.